smoke-tests/configure.gradle

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Google LLC
```

Comment:
In JS SDK, we have a script that can fix the license header (or add if missing). It's also hooked with the git, so that whenever someone executes a git push, the script will get triggered automatically, fix the header and generate a new commit. We may consider to have something similar in Android SDK.